style/servo/selector_parser.rs

```diff
@@ -61,10 +61,15 @@ pub enum PseudoElement {
     ServoAnonymousTableRow,
     ServoTableGrid,
     ServoTableWrapper,
+
+    // Implemented pseudos. These pseudo elements are representing the
```

Comment:
Does "implemented pseudos" refer to element-backed pseudo-elements (https://drafts.csswg.org/css-pseudo-4/#element-like)?

I think DetailsSummary and DetailsContent also belong to that category.

i would be better to group them together.

Reply:
Following Firefox's definition, it is different. And, the detail content and detail summary pseudo elements defined in Servo precedes the `::details-content` defined in the spec. 

Not to mention we are not using these pseudo elements anymore for the display of details element. Though, a cleanup PRs for these could probably follow in the future.